File: introduction-ws-client/studentDto.py
class StudentDto:
    def __init__(self, id, name, enrolled):
        self.__name = name
        self.__id = id
        self.__enrolled = enrolled

    @property
    def id(self):
        return self.__id

    @property
    def name(self):
        return self.__name

    @property
    def enrolled(self):
        return self.__enrolled

    def __str__(self):
        return f"{self.__id} - {self.__name} - {self.__enrolled}"

File: api/src/main/kotlin/tsi/ws/lucaschfonseca/introduction/api/controller/dto/StudentDto.kt
package tsi.ws.lucaschfonseca.introduction.api.controller.dto

import tsi.ws.lucaschfonseca.introduction.api.model.Student
import java.util.stream.Collectors

class StudentDto(student: Student) {
    val id: Long
    val name: String
    val isEnrolled: Boolean

    init {
        id = student.id
        name = student.name
        isEnrolled = student.isEnrolled
    }

    override fun toString(): String {
        return "StudentDto{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", enrolled=" + isEnrolled +
                '}'
    }

    companion object {
        @JvmStatic
        fun map(students: List<Student>): List<StudentDto> {
            return students.stream().map { student: Student -> StudentDto(student) }.collect(Collectors.toList())
        }
    }
}